device/tt_silicon_driver.cpp

```diff
@@ -224,6 +232,14 @@ struct TTDevice : TTDeviceBase
     TTDevice() = default;
 
     void reset() {
+        if (arch == tt::ARCH::BLACKHOLE && bar2_uc != nullptr && bar2_uc != MAP_FAILED) {
+            // Disable ATU.
```

Comment:
Does this disable all regions that were enabled or just index zero?

Reply:
Oh - I see that `initialize_pcie_devices` ensures only a single channel in the Blackhole case.  May be worth leaving a TODO here (or just implement it) so that when the code grows the ability to use multiple channels, all corresponding iATU regions are disabled here rather than just index 0.

Reply:
I'd rather leave a comment. I'm new in this codepath, so any code change however small can take me significant time. Not sure that mapping more than one channel was used on wormhole, so I'd say this is not a priority.